pkg/controller/nodefeaturediscovery/nodefeaturediscovery_controller.go

```diff
@@ -60,6 +60,11 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
+	// Watch for changes to ConfigMap nfd-worker.conf
+	if err = c.Watch(&source.Kind{Type: &corev1.ConfigMap{}}, &handler.EnqueueRequestForObject{}); err != nil {
+		return err
+	}
```

Comment:
Prolly a stupid question, but how does this change prevent the controller from overwriting the ConfigMap? Isn't the `Reconcile()` run whenever any of the watched resources are modified(?)

Reply:
It is a good question, it will not prevent it. If you e.g. delete the DaemonSet the operator will overwrite and update the CM since a watched resource changed the the reconcile loop kicks in.